src/tools/genpolicy/Cargo.toml

```diff
@@ -5,7 +5,7 @@
 
 [package]
 name = "genpolicy"
-version = "0.1.0"
+version = "3.2.0-1"
```

Comment:
```suggestion
version = "3.2.0.azl0.genpolicy1"
```

Reply:
Version field will only accept semantic versioning. `3.2.0.azl0.genpolicy1` is not accepted by cargo
![image](https://github.com/microsoft/kata-containers/assets/30801614/d0fbed45-17f5-4101-8324-d75661ee17f2)

But funnily, it will take `3.2.0-azl0.genpolicy1` (with a dash instead of dot in the middle). Updating to that